CGPs/cgp-0061.md

```diff
@@ -1,71 +1,49 @@
 ---
-cgp: <to be assigned>
+cgp: 61
 title: Governance-Owned cUSD<->USDC Liquidity
 date-created: 2022-08-03
 author: cLabs & Mento
-status: <DRAFT>
```

Comment:
Do not allow <> around the status values or they'll fail equality checks with the other values. And more boardly, <> chars in strings can cause problems when rendering them on the web so recommend avoiding them entirely for content in this repo